crates/rpc/rpc/src/eth/api/call.rs

```diff
@@ -40,6 +40,93 @@ where
         BlockReaderIdExt + ChainSpecProvider + StateProviderFactory + EvmEnvProvider + 'static,
     Network: NetworkInfo + Send + Sync + 'static,
 {
+    /// Executes   complex RPC calls to Ethereum nodes
+
+    pub async fn eth_multicall_v1(
+        &self,
+        multicall_bundle: Bundle,
+        state_context: Option<StateContext>,
+        mut state_override: Option<StateOverride>,
+    ) -> EthResult<Vec<EthCallResponse>> {
```

Comment:
this is just the same as call_many?

so atm we don't need this function and can reuse call_many instead

Reply:
> this is just the same as call_many?
> 
> so atm we don't need this function and can reuse call_many instead

hey ty for your review, you means i should directly call the function call_many inside this function ?

Reply:
yes, at least for now

Reply:
> yes, at least for now

ty so much, very greatful from your part all these review. should be fine now